mffpy/writer.py

```diff
@@ -69,7 +70,10 @@ def write(self):
 
         # clean up in case of overwrite
         for file in old_files:
-            remove(join(self.mffdir, file))
+            try:
+                rmtree(join(self.mffdir, file))
+            except NotADirectoryError:
+                remove(join(self.mffdir, file))
```

Comment:
Why not just remove `self.mffdir`?

Reply:
> This sounds like a good solution. We could add a cleanup step to Writer.write which deletes any files in the output directory that are not in Writer.files.

As I am not familiar with `mffpy` and all the possible content for `self.mffdir`, I went for what was written here.

Reply:
You would go for `rmtree(self.mffdir)` as clean-up step if `overwrite` is set to True, instead of the file by file removal? That works for me to and is simpler.

Reply:
Yes, I think this might be fine.  On windows in Python 3.8 there seems to be some relevant change:

> Changed in version 3.8: On Windows, will no longer delete the contents of a directory junction before removing the junction.

If you can, please have someone test this on a windows system.

Reply:
> Yes, I think this might be fine.

Current version, or `rmtree(self.mffdir)` as a clean-up step?

For the change in 3.8, I'm not sure about what it means. If the junction is removed, then all the content is also removed? So it should not change anything? Do you have a CI/workflow running on windows?

Reply:
I can add a test where I create a directory with subdirectories in `self.mffdir` and test if it does get deleted.

Reply:
If we're going this route, I would put the cleanup step in the `create_directory` method. If `overwrite` is `True`, `create_directory` removes all contents of the existing MFF directory, or creates a new one if it doesn't exist yet.

Reply:
done